mod-finance/schemas/budget.json

```diff
@@ -48,34 +49,50 @@
       "description": "The amount currently expended for this budget",
       "type": "number"
     },
+    "unavailable" : {
```

Comment:
This can be done only if we move these three properties to a separate schema definition (json object with three properties) and `ref` it by other schemas using some common property e.g. `summary`. 
As alternative we could use for such cases [allof](https://json-schema.org/understanding-json-schema/reference/combining.html#allof) notion. However it does not support `additionalProperties: false`
> Unfortunately, now the schema will reject _everything_. This is because the [Properties](https://json-schema.org/understanding-json-schema/reference/object.html#additionalproperties) refers to the entire schema. And that entire schema includes no properties, and knows nothing about the properties in the subschemas inside of the [allOf](https://json-schema.org/understanding-json-schema/reference/combining.html#allof) array.
> 
> This shortcoming is perhaps one of the biggest surprises of the combining operations in JSON schema: it does not behave like inheritance in an object-oriented language. There are some proposals to address this in the next version of the JSON schema specification.

Reply:
It would be great, but `allocated` is required for the budget, and for the rest of the schemes is not.